api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -85,4 +86,44 @@ default void updateBillingStatus(long workspaceId, BillingStatus status) {
 
     Long getWorkspaceCount();
   }
+
+  @Query(
+      "SELECT\n"
+          + "  w.billingAccountType,\n"
+          + "  w.billingStatus,\n"
+          + "  w.cdrVersion.cdrVersionId AS cdrVersionId,\n"
+          + "  w.creationTime,\n"
+          + "  w.creator.userId AS creatorId,\n"
+          + "  w.disseminateResearchOther,\n"
+          + "  w.lastAccessedTime,\n"
+          + "  w.lastModifiedTime,\n"
+          + "  w.name,\n"
+          + "  w.needsResearchPurposeReviewPrompt AS needsRpReviewPrompt,\n"
```

Comment:
The projection, BigQuery, and DTO column names always agree with the MySql column name. Frequently that doesn't agree with the property exposed on the entity object so we have these `as` conversions.